### DISCUSSION TITLE
Are the type_map.raw files in different data are the same?

### DISCUSSION THREAD

**Discussion by gswylq** [deepmodeling/deepmd-kit#2793] [Q&A] [answered]
Hello,

Are the type_map.raw files in different data the same? Whether the order of element symbols must be the same? If it is different, will it affect the calculation result?
Thanks！

Q Li

**Comment by njzjz** [comment, marked as answer]
> Are the type_map.raw files in different data the same? Whether the order of element symbols must be the same?

No.
